docs/content/tutorial/backups.md

```diff
@@ -123,6 +128,68 @@ kubectl apply -k kustomize/s3
 
 Watch your cluster: you will see that your backups and archives are now being stored in S3!
 
+### Using an AWS-integrated identity provider and role
+
+If you deploy PostgresClusters to AWS Elastic Kubernetes Service, you can take advantage of their 
```

Comment:
FYI it's not specific to EKS; anyone can run AWS IRSA. (even if you run your own kubernetes cluster)

It requires running https://github.com/aws/amazon-eks-pod-identity-webhook, which is automatically run *for* you in EKS; but that doesn't make it a pre-requisite.